src/dawn/IIR/Stencil.cpp

```diff
@@ -100,20 +116,21 @@ Stencil::Stencil(StencilInstantiation& stencilInstantiation,
 
 std::unordered_set<Interval> Stencil::getIntervals() const {
   std::unordered_set<Interval> intervals;
-  for(const auto& multistage : multistages_)
-    for(const auto& stage : multistage->getStages())
-      for(const auto& doMethod : stage->getDoMethods())
+  for(const auto& multistage : children_)
```

Comment:
inconsistency, `childern_` vs `getChildren()`

Reply:
can you explain why would you use getChildren() over children_ ? 
The usual pattern was using getChildren() to have access to children_ from outside the class, but from the class we can just access the member.

Reply:
see comment above